test/with_tmp_db

```diff
@@ -1,67 +1,111 @@
 #!/usr/bin/env bash
-
 set -euo pipefail
 
+export KEEP_TMP=${KEEP_TMP:-""}
+
 usage() {
-    cat << EOF
+  cat << EOF
 
-USAGE: $0 <command>
+USAGE: $0 COMMAND
 
-Runs the given command with the POSTGREST_TEST_CONNECTION environment variable
+Runs the given COMMAND with the POSTGREST_TEST_CONNECTION environment variable
 set to a temporary database that is ready for running the PostgREST test suite.
 
-You'll need to have the Postgres binaries 'initdb' and 'pg_ctl' on your PATH.
+You'll need to have the Postgres binaries 'initdb', 'pg_ctl' and 'psql' on your
+PATH.
 
 Example:
 
-    $0 stack test
+    $0 cabal v2-test
```

Comment:
For users that won't be able to use nix-shell, I think the `stack test` example should be kept. It's working really good with stack.

Reply:
Maybe something like:
```suggestion
    $0 stack v2-test
    
    ## Or for when inside nix-shell
    $0 cabal v2-test
```

Reply:
Good idea, thanks for the fix!